zipkin-server/src/main/java/zipkin2/server/internal/ZipkinServerConfiguration.java

```diff
@@ -75,7 +77,7 @@
       if (httpCollector != null) sb.annotatedService(httpCollector);
       if (healthController != null) sb.annotatedService(healthController);
       // Redirects the prometheus scrape endpoint for backward compatibility
-      sb.service("/prometheus", new RedirectService("/actuator/prometheus"));
+      sb.service("/prometheus", new PrometheusExpositionService(prometheusRegistry));
```

Comment:
If I understand correctly, we just have now 2 implementations of /prometheus. actuator is handling the other one

Reply:
Yeah - I think that's what #2788 meant but maybe I misunderstood :)

Reply:
Ah think I missed latest comment - so you think we should use this service on `/actuator/prometheus` too?

Reply:
well right now we aren't yanking actuator, but when we get around to it, we should mount it. that or disable the autoconfig and mount it now.

Reply:
Went ahead and disabled the actuator/prometheus endpoint and mount it now.